### PR TITLE
Update classes-and-objects.md

### PR DESCRIPTION
**Pull request by delta-nry**

### CODE REVIEW

**Review by BillWagner — CHANGES_REQUESTED** [review]
Thanks for the contribution @delta-nry. We appreciate it.

Overall, what you've done makes this more clear and easier to read. I left one suggested improvement. If you can make that change, I'll merge this in.

**Comment by delta-nry** [comment]
Hi @BillWagner, I could not find the suggested improvement that you told me about. Could you tell me more about the improvement that you would like to add?

File: docs/csharp/tour-of-csharp/classes-and-objects.md [hunk 1]
```diff
@@ -196,7 +196,7 @@ The previous four classes can be used to model arithmetic expressions. For examp
 
 [!code-csharp[ExpressionExample](../../../samples/snippets/csharp/tour/classes-and-objects/Program.cs#L40-L43)]
 
-The `Evaluate` method of an `Expression` instance is invoked to evaluate the given expression and produce a `double` value. The method takes as an argument a @`Dctionary` that contains variable names (as keys of the entries) and values (as values of the entries). The `Evaluate` method is a `virtual abstract` method, meaning that non-abstract derived classes must override it to provide an actual implementation.
+The `Evaluate` method of an `Expression` instance is invoked to evaluate the given expression and produce a `double` value. The method takes a `Dictionary` argument that contains variable names (as keys of the entries) and values (as values of the entries). The `Evaluate` method is an abstract method, and because abstract methods are implicitly virtual methods, non-abstract classes derived from `Expression` must override `Evaluate` and provide an actual implementation for the method.
```
Comment (svick) [inline docs/csharp/tour-of-csharp/classes-and-objects.md]: This now seems to imply that non-abstract classes must override all `virtual` methods, which is not true. I would remove the part about "virtual" altogether:

> The `Evaluate` method is an abstract method, which means that non-abstract classes derived from `Expression` must override `Evaluate` and provide an actual implementation for the method.

**Comment by BillWagner** [comment]
@delta-nry It looks like I didn't save my comment in the review. Strange.  My comment was almost exactly like the comment left by @svick If you can make that change, we'll merge this.

**Review by BillWagner — APPROVED** [review]

**Comment by BillWagner** [comment]
Thanks again for contributing @delta-nry 
The changes look good.

As soon as the build passes, I'll :shipit:

**Pull request merged**